utils/restUtils.js

```diff
@@ -1,8 +1,22 @@
-import { AuthenticationError, UserInputError, ServerError } from 'apollo-server-express';
+import { AuthenticationError, UserInputError, ApolloError } from 'apollo-server-express';
+import logger from './logger';
 /*
 convert the REST status codes to GQL errors, or return the response if passing
 */
 export const restErrorResponseHandler = async response => {
+  // Generic handle 5xx errors
+  if (response.status >= 500 && response.status <= 599) {
+    let responseBody;
+    try {
+      responseBody = await response.json();
```

Comment:
since we're not expecting to parse `responseBody` for anything, I think `.text()` is safer. The only reason `.json()` would fail is because rest APIs are not guaranteed to return json, which is an assumption we can safely bypass with `.text()`.